index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lisandro-Subrat</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="container">
+      <div id="header">
+        <h1 id="title">survey</h1>
+        <p id="description">Thanks for filling the survey!</p>
+      </div>
+
+      <form id="survey-form">
+        <div class="form-control">
+          <label id="name-label">Name:</label>
+          <input
+            type="text"
+            placeholder="Enter your name"
+            name="name"
+            id="name"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label id="email-label">Email:</label>
+          <input
+            type="email"
+            placeholder="Enter your email"
+            name="email"
+            id="email"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label id="number-label">age:</label>
+          <input
+            type="number"
+            placeholder="Enter your age"
+            name="age"
+            id="number"
+            min="10"
+            max="100"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label>Dropdown options:</label>
+          <select name="Dropdown" id="dropdown">
+            <option disabled="" selected="" value="">Select an option</option>
+            <option value="option1">Option 1</option>
+            <option value="option2">Option 2</option>
+            <option value="option3">Option 3</option>
+            <option value="option4">Option 4</option>
+          </select>
+        </div>
+        <div class="form-control">
+          <h2>Radio Options :</h2>
+          <input type="radio" id="age1" name="age" value="30" />
+          <label>option 1</label><br />
```

Comment:
- [ ] I highly recomend you to put  labels before the checkbox, so the user knows better what he is choosing, otherwise, in a longer form, everyone can get confused :sweat_smile:
```suggestion
          <label>option 1</label><br />
          <input type="radio" id="age1" name="age" value="30" />
```
#1